src/profile/UitpasCard/UitpasCard.tsx

```diff
@@ -14,44 +17,52 @@ type TProps = {
 
 const UitpasCard: FC<TProps> = ({ passHolder }) => {
   const { width } = useWindowDimensions();
+  const [logOutModalVisible, toggleLogOutModalVisible] = useToggle(false);
```

Comment:
Copy-paste ? 🤔 Should be something like `cardModalVisible`

Reply:
![image](https://user-images.githubusercontent.com/90620650/208449610-556aa26c-2549-410b-b466-876245ce51a3.png)